src/github/client/github-client.ts

```diff
@@ -104,7 +107,7 @@ export default class GitHubClient {
 	}
 
 	private async graphql<T>(query: string, variables?: Record<string, string | number | undefined>): Promise<AxiosResponse<GraphQlQueryResponse<T>>> {
-		const response = await this.axios.post<GraphQlQueryResponse<T>>(this.githubInstallationId.githubBaseUrl +"/graphql",
+		const response = await this.axios.post<GraphQlQueryResponse<T>>("/graphql",
```

Comment:
I've realised that we set the same base URL to the client itself already.

We have integration tests which cover this cahnge